bsmetadata/hydra_configs/v2.yaml

```diff
@@ -37,33 +37,15 @@ data_config:
         attributes_to_keep:
         - class
         - id
-        txt_max_chr_len: 128.0
+        txt_max_chr_len: 0
         txt_min_chr_len: -.inf
-        tags_exceptions_to_txt_max_min_chr_len:
-        - tabletr
-        - th
-        - td
-        - caption
-        - colgroup
-        - thead
-        - tfoot
-        - tbody
-        - li
-        - ol
-        - ul
-        - span
+        tags_exceptions_to_txt_max_min_chr_len: null
```

Comment:
I'm not entirely sure about the yaml syntax here, but the idea is that I don't want to define exceptions (so in the code this should result in an empty list I believe)